pkg/oc/admin/diagnostics/diagnostics/cluster/network/in_pod/util/util.go

```diff
@@ -208,10 +208,12 @@ func Execute(factory kcmdutil.Factory, command []string, pod *kapi.Pod, in io.Re
 			Namespace:     pod.Namespace,
 			PodName:       pod.Name,
 			ContainerName: pod.Name,
-			In:            in,
-			Out:           out,
-			Err:           errOut,
-			Stdin:         in != nil,
+			IOStreams: genericclioptions.IOStreams{
```

Comment:
@juanvallejo we need to wire this bits properly as well.

Reply:
ack